subsys/nrf_security/configs/psa_unsupported_ciphers_check.h

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ * Copyright (c) since 2020 Oberon microsystems AG
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/* Currently Unsupported Algorithms */
```

Comment:
Let's add a comment so we don't make changes to this file.

/*
* The following code is imported from Oberon and should not be modified.
*/